alvr/session/src/settings.rs

```diff
@@ -1137,6 +1162,18 @@ Because of runtime limitations, this option is ignored when body tracking is act
     ))]
     pub multimodal_tracking: bool,
 
+    #[schema(flag = "steamvr-restart")]
+    #[schema(strings(
+        help = r"If it is activated left controller will be used as fake tracker in steamVR"
+    ))]
+    pub use_left_as_tracker: Switch<ControllersFakeTrackingBinding>,
+
+    #[schema(flag = "steamvr-restart")]
+    #[schema(strings(
+        help = r"If it is activated right controller will be used as fake tracker in steamVR"
+    ))]
+    pub use_right_as_tracker: Switch<ControllersFakeTrackingBinding>,
```

Comment:
The help string should use active voice i.e. "Use right controller as fake tracker"